docs/products/kafka/kafka-connect/concepts/list-of-connector-plugins.rst

```diff
@@ -0,0 +1,104 @@
+List of available Apache Kafka Connect connectors
+=================================================
+
+The following Apache Kafka Connect connectors are currently available, and can
```

Comment:
Do we have to use "Apache Kafka Connect " in "Apache Kafka Connect connectors", when so many words are repeating, makes it difficult to read and process.

Reply:
I know, but it's the used wording in this space, not sure how and if we should avoid it